tensorboard/webapp/metrics/views/card_renderer/histogram_card_test.ts

```diff
@@ -385,6 +411,7 @@ describe('histogram card', () => {
           start: {step: 18},
           end: {step: 20},
         });
+        store.overrideSelector(selectors.getMetricsRangeSelectionEnabled, true);
```

Comment:
Can we just put this in the top level describe?

Reply:
It's not `true` for every test but I think you're right that it makes more sense in a `beforeEach`